src/utilities/api/auth.js

```
@@ -0,0 +1,143 @@
+
+   if (typeof email
+      === "string"
+      && typeof password
+      === "string"
+      && email.length
+      > 0
+      && password.length
+      > 0
+      && username.length
+      > 0) {
+   }
```

ESLint

[error] Empty block statement.

(no-empty)